### DISCUSSION TITLE
Merging Multiple IFCs? (more than 2)

### DISCUSSION THREAD

**Discussion by ghost** [IfcOpenShell/IfcOpenShell#4202] [General]
Hello

I am hoping someone is able to help me.

I was wondering if anyone knows if it is possible to merge two or more IFCs with BlenderBIM??

**Comment by Moult** [comment]
Use [IfcPatch](https://blenderbim.org/docs-python/ifcpatch.html) with the [MergeProject](https://blenderbim.org/docs-python/autoapi/ifcpatch/recipes/MergeProject/index.html) recipe.
